index.html

```diff
@@ -0,0 +1,107 @@
+  <link rel="stylesheet" href="./styles/navbar.css">
+  <link rel="stylesheet" href="./styles/general.css">
+  <link rel="stylesheet" href="./styles/headline.css">
+  <link rel="stylesheet" href="./styles/program.css">
+  <link rel="stylesheet" href="./styles/musicians.css">
+  <link rel="stylesheet" href="./styles/footer.css">
+</head>
+<body>
+  <nav class="navBar">
+    <button class="hambugerMenu"  id="hamburgerMenu">
+      <img class="hambugerMenuImg" src="./assets/navbar/Icon - Menu.png" alt="menu">
+    </button>
+    <ul class="navbarList">
+      <li class="navbarListItem navbarFirst">
+        <a href="./index.html"><img class="orchestraIcon" src="./assets/navbar/orchestraIcon.jpg" alt="Cordoba Spring Festival 2023 Icon"></a> 
+      </li>
+      <li class="navbarListItem"><a class="navbarLink" href="./about.html">About</a></li>
+      <!--I believe this anchors should take the user to other pages, just like the about anchor above this comment does,
+      but for the purpose of this projects, does pages are not done yet. so the anchors are empty, to be compleated on a 
+      later date-->
+      <li class="navbarListItem"><a class="navbarLink">Program</a></li>
+      <li class="navbarListItem"><a class="navbarLink">Join</a></li>
+      <li class="navbarListItem"><a class="navbarLink">Sponsor</a></li>
+      <li class="navbarListItem"><a class="navbarLink">News</a></li>
+      <li class="navbarListItem lastItem"><a>Spring Festival</a></li>
+    </ul>
+  </nav>
+  <main>
+    <section class="headline">
+      <div class="headlineDiv">
+        <p class="miniTitle">"Hello! Music World"</p>
+        <h1 class="pageTitle">
+          CORDOBA CLASICAL<br/> SPRING FESTIVAL <br/>2023
+        </h1>
+        <p class="titleDescription">
+          Immerse yourself in the melodies of Beethoven, Mozart, 
+          and Handel at the Cordoba Spring Festival.
+          Experience the beauty of classical music in a captivating setting.
+        </p>
+        <h2 class="dateOfConcert">
+          2023.7.7(FRI) ~ 8(SAT)
+        </h2>
+        <p class="places">
+          @ Cordoba Real Theater, Art Center Güemes and more
+        </p>
+      </div>
+    </section>
+    <section class="program" id="sectionLink1">
+      <h3 class="programSectionTitle">
+        Main Program
+      </h3>
+      <div class="musicDiv" id="musicProgram">
+      </div>
+      <div>
+        <button class="joinConcertBtn">
+          Join Cordoba Clasical Spring Festival 2023
+        </button>
+      </div>
+    </section>
+    <section class="featuredMusicians" id="sectionLink2">
+      <h3 class="director">
+        Musicians
+      </h3>
+      <div class="cardsDiv" id="musiciansCardsGroup">
+      </div>
+      <button class="moreBtn" id="moreCardsBtn">
+        MORE
+      </button>
+    </section>
+    <footer class="homeFooter">
+      <article class="sponsors">
+        <div class="sponsorTitleDiv">
+          <p class="sponsorTitle">Partner</p>
```

Comment:
- [ ] Kindly add the red line under the title as in the provided design to make your website a copy of the provided design
![Screenshot from 2023-06-29 02-31-35](https://github.com/IsmaelMastronardi/Capstone1/assets/91142432/c7ea35ea-335c-414e-bc1d-bb2a910681be)